audible.py

```diff
@@ -339,6 +339,7 @@ def get_album_info(self, asin):
             "composer": narrators, "grouping": content_group_description,
             "genre": genres, "series_name": series_name, "series_position": series_position,
             "comments": description, "data_source": self.data_source, "subtitle": subtitle,
+            "catalognum": asin
```

Comment:
Album level seems right. But then several other common_attributes should be moved to album level as well.

asin, album_sort, grouping, series, series_position, comments, subtitle. All of these seem album level to me. I'm going mostly by this.

https://github.com/beetbox/beets/blob/404229b84507076c8f16f65b6f948739ec9dfb72/test/helper.py#L582-L587

Reply:
Ah, those fields you mentioned are set at the track level so that Beets applies them to each track.

I'm suggesting that for cataloguenum to be at the album level since i believe it isn't written to ID3 tags.

Reply:
Catalognum is written to tags. That's what happens in my music collection that I tag with beets.